ServiceCapacityTracker/ServiceCapacityTrackerFunction/src/main/java/com/amazon/vas/ServiceCapacityTracker/Model/GetMerchantAggregatedDetailsOutput.java

```diff
@@ -0,0 +1,12 @@
+package com.amazon.vas.ServiceCapacityTracker.Model;
+
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+
+@Builder
+@Value
+public class GetMerchantAggregatedDetailsOutput {
```

Comment:
all models which are input and output of other services like SPIN, VOS etc. move those in respective sub package inside model package..